New_File2

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+from utils.constants import (FB_URL, FIRST_NAME, SUR_NAME, EMAIL_ADDRESS, NEW_PASSWORD)
+
+class SignUpForm():
+     """This class contains functions"""
+    def find_element(self, css_selector='')
+        if css_selector:
+            return self.driver.find_element_by_css_selector(css_selector)
+
+    def clear_value_of_element(self, css_selector):
+        element = self.find_element(css_selector=css_selector)
+        element.clear()
+        return element
+
+    def enter_credentials_in_signUp_form(self, first_name='', last_name='',
```

Comment:
@ruttaba Rename function and fix the word "signUp". Python function names should not contain uppercase letters. They should be in snake case. Please read about snake case and naming conventions in Python.